submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/filesystem/SubmissionFilesPermissions.kt

```diff
@@ -0,0 +1,16 @@
+package ac.uk.ebi.biostd.persistence.service.filesystem
+
+import java.nio.file.attribute.PosixFilePermission
+import java.nio.file.attribute.PosixFilePermissions
+
+enum class SubmissionFilesPermissions(val stringValue: String) {
```

Comment:
it looks to me that this class will be easier to model as a simple data class with the two `RW_RW____` constants values (files and folder).

Reply:
You're right, this seems like an overhead. I just went ahead and removed the specific enum class and passed down the corresponding file/folder permissions. I think it's much clearer.